src/Global/GlobalComponents/Widgets/KineticForm.jsx

```diff
@@ -38,6 +46,7 @@ export const KineticForm = props => {
             onLoaded={handleLoaded()}
             onUpdated={handleRedirect()}
             onCompleted={handleRedirect()}
+            components={{Pending}}
```

Comment:
Let's spread anything coming out of the props as well so they aren't ignored if they come in as a prop. This would look like `components={{Pending, ...props.components}}`